data/Templates/Hl7v2/Resource/_Location.liquid

```diff
@@ -7,17 +7,17 @@
         {% include 'DataType/PL' PL: PV1.3 -%}
         {% include 'DataType/PL' PL: PV1.6 -%}
         {% include 'DataType/PL' PL: PL -%}
-        {% include 'DataType/PL' PL: PR1.23 -%}
-        {% include 'DataType/PL' PL: PRT.9 -%}
-        {% include 'DataType/XONLocation' XON: ORC.21 -%}
+        {% include 'DataType/PL' PL: PR1.23.Repeats[0] -%}
+        {% include 'DataType/PL' PL: PRT.9.Repeats[0] -%}
+        {% include 'DataType/XONLocation' XON: ORC.21.Repeats[0] -%}
         "address":
         {
-            {% include 'DataType/XAD' XAD: ORC.22 -%}
+            {% include 'DataType/XAD' XAD: ORC.22.Repeats[0] -%}
```

Comment:
consider to move repeats into XAD

Reply:
Here we don't actually use the repeatable identifiers but just take the first element of the array, so the `ORC.22.Repeats[0] ` is just a single identifier, represents an unitary data.